Drive/rover_drive_system.hpp

```diff
@@ -92,6 +94,22 @@ class RoverDriveSystem
     }
   };
 
+
+
+  bool isSyncedWithMissionControl()
+  {
+      int expectedHeartbeat = heartbeat_count_+1;
```

Comment:
nitpicky but this should be `expected_heartbeat` as described in [style guide ](https://google.github.io/styleguide/cppguide.html#Variable_Names)